seed-bucket.js

```diff
@@ -0,0 +1,53 @@
+const faker = require('faker');
+const fs = require('fs');
+const AWS = require('aws-sdk');
+const request = require('request');
+
+const s3 = new AWS.S3({
+  accessKeyId: ``, //deleted but stored;
+  secretAccessKey: `` //deleted but stored;
+});
```

Comment:
👏 good job in remembering, otherwise $--! You could also have saved them in a separate file in variables, import that file here for access, and ensured that is git-ignored.